src/core/partition.c

```diff
@@ -96,15 +96,15 @@ QuicPartitioGetStatelessRetryKey(
     CxPlatCopyMemory(
         RawKey,
         MsQuicLib.BaseRetrySecret,
-        sizeof(MsQuicLib.BaseRetrySecret));
+        CxPlatKeyLength(MsQuicLib.RetryAeadAlgorithm));
```

Comment:
Instead of recalculating length every time (I know it's just a switch statement), we should optimize for perf and save the length.